### PR TITLE
Page for the office, also resolved lint complaint

### PR DESCRIPTION
**Pull request by brettallenyo**
Created subdirectory for everything related to the office.

This subdirectory consists of the necessary html, css, and js files in order to make a page that has a button that the user can click to generate a random office quote. Fun stuff!

Additionally, lint complained about the js file in the main webapp directory so i removed that because it was not being used anyways.

https://screenshot.googleplex.com/MR6EP4m82Hx

### CODE REVIEW

File: portfolio/src/main/webapp/theoffice/theoffice.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <script src="theoffice.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <div>
+        <a href="/../index.html" class="button button-black">Home</a>
+      </div>
+      <h1>Click here to get a random Office quote:</h1>
+      <a class="button button-purple" onclick="addRandomGreeting()">Click me!</a>
+      <h3 id="greeting-container">.</h3>
```
Comment (chen-dawn) [inline portfolio/src/main/webapp/theoffice/theoffice.html:17]: remove extra dot "." in this div? Unless this is intended.

File: portfolio/src/main/webapp/theoffice/theoffice.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <title>The Office</title>
+    <link rel="stylesheet" href="theoffice.css" />
+    <link rel="stylesheet" href="/../buttons.css" />
+    <script src="theoffice.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <div>
+        <a href="/../index.html" class="button button-black">Home</a>
+      </div>
```
Comment (chen-dawn) [inline portfolio/src/main/webapp/theoffice/theoffice.html:14]: It's probably better to move this button above the `<div id="content">`
So a structure like:
```
<body>
  <div id="navigation">
    Navigation stuff
  </div>
  <div id="content">
    Actual page content
  </div>
</body>
```

File: portfolio/src/main/webapp/theoffice/theoffice.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#content {
+  margin: auto;
+  text-align: center;
+  width: 650px;
+  height: 300px;
+  background-color: #ffffff;
```
Comment (chen-dawn) [inline portfolio/src/main/webapp/theoffice/theoffice.css:6]: sort CSS alphabetically

File: portfolio/src/main/webapp/theoffice/theoffice.html (hunk 1 above)
Comment (brettallenyo) [reply portfolio/src/main/webapp/theoffice/theoffice.html:17]: For some reason lint complains about the empty h3, so i have to put some sort of text in there. Any ideas of how I can resolve this?

Comment (brettallenyo) [reply portfolio/src/main/webapp/theoffice/theoffice.html:17]: Update, I just disabled the warning for now inside the html, there might be a way to bypass it by simply making it some other value like <p> with some styling. But its not a huge deal

**Pull request merged**